frontend/public/components/droppable-edit-yaml.tsx

```diff
@@ -1,12 +1,14 @@
 import withDragDropContext from './utils/drag-drop-context';
 import * as React from 'react';
-import {EditYAML} from './edit-yaml';
+
+import { EditYAML } from './edit-yaml';
 import { NativeTypes } from 'react-dnd-html5-backend';
 import { DropTarget } from 'react-dnd';
 
 // Maximal file size, in bytes, that user can upload
 const maxFileUploadSize = 4000000;
 const fileSizeErrorMsg = 'Maximum file size exceeded. File limit is 4MB.';
+const fileTypeErrorMsg = 'Binary file detected. Edit text based YAML files only.';
```

Comment:
I wonder if there's a way to default to `.yaml` / `.yml` / `.json` file extensions in the file browser.

Reply:
@spadgett possibly, but we may block files that have valid yaml text that have no extension or different extensions.

Reply:
I think it's a suggestion, and it's possible in most platforms to pick any file. I'd have to look at the API though

Reply:
Specifically, I'm suggesting the `FileInput` component accepts an optional `accept` prop that it uses on the `<input type="file">` element

https://stackoverflow.com/questions/4328947/limit-file-format-when-using-input-type-file

Reply:
@spadgett this isn't a file chooser. The user drags a file from any window into the editor, so there is no filtering of file options before the drop happens.

Reply:
@jcaianirh Right, I forgot that we didn't include the file input as well in the Import YAML page.